packages/trpc/server/routers/viewer/routing-forms/findTeamMembersMatchingAttributeLogicOfRoute.handler.ts

```diff
@@ -164,7 +165,14 @@ export const findTeamMembersMatchingAttributeLogicOfRouteHandler = async ({
     });
   }
 
-  const eventType = await EventTypeRepository.findByIdIncludeHostsAndTeam({ id: eventTypeId });
+  const eventType = eventTypeId
+    ? await EventTypeRepository.findByIdIncludeHostsAndTeam({ id: eventTypeId })
+    : await EventTypeRepository.findBySlugAndUserIdOrTeamId({
+        // Assume that the slug was saved in the format of team/{teamName}/{eventSlug}?query
+        slug: eventTypeUrl.split("/")[2].split("?")[0],
```

Comment:
This is very fail prone. Accessing an index 2 of a URL that might not have will crash. Are we certain the URL is validated somewhere else?